<launch>
    <!-- IMU NODE -->
    <node pkg="barc" type="imu_data_acquisition.py" name="imu_node" >
        <param name="port" value="/dev/ttyACM0" />
    </node>

    <!-- ARDUINO NODE -->
    <!--  * encoders and ultrasound sensors -->
    <node pkg="rosserial_python" type="serial_node.py" name="arduino_node" >
        <param name="port" value="/dev/ttyUSB0" />
    </node>

	<!-- OPEN LOOP MANUEVERS -->
    <node pkg="barc" type="controller_circular.py" name="controller" >
        <!--  SELECTION -->
        <param name="user" value="jgon13" />
       <param name="experiment_sel" type="int" value="0" />

        <!-- PROPERTIES -->
        <param name="v_x_pwm" type="int" value="97" />
        <param name="steering_angle" type="int" value="25" />
		<param name="t_exp" type="int" value="5" />
        <param name="t_0" type="int" value="2" />
    </node>

    <!-- Record the experiment data -->
    <node pkg="rosbag" type="record" name="rosbag_record"
        args=" --all" />

</launch>
